src/components/updatemenus/draw.js

```diff
@@ -253,11 +254,19 @@ function drawButtons(gd, gHeader, gButton, menuOpts) {
 
         exit.transition()
             .attr('opacity', '0')
-            .remove();
+            .remove()
+            .each('end', function() {
```

Comment:
![gifrecord_2017-01-31_143701](https://cloud.githubusercontent.com/assets/6675409/22481325/d422f080-e7c2-11e6-9a66-937e34a160a3.gif)

This makes the scrollbox disappear after the button exit transition is completed (you can chain `.delay(1000)` to the transition to make the problem easier to detect). 

I would prefer making the scrollbox disappear at the same time as the buttons.